CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Changes since v2.13
 - Check file extensions ignoring case (#2392)
 - Fixed `java -jar rems.jar help`. See [docs/installing-upgrading.md](docs/installing-upgrading.md)
 - Inconsistencies organization owner logic. (#2441)
+- Fix aria-required and default link contrast (#2431)
```

Comment:
how about something a bit clearer like  "Improve accessibility by adding ARIA required attributes and increasing contrast of links"